alphagov/paas-developer-docs
# Redirect readthedocs to docs.cloud.service.gov.uk

## What

Readthedocs doesn't allow redirections, or custom themes that would allows us to deploy a simpler solution, so this replaces every page in the documentation with a block of text linking to the correct page in the new documentation.

Automatic redirection is possible because RTD allows you to include custom javascript, so this simply grabs the link in the page and changes the window location after 5 seconds.

## How to review

1. Install `mkdocs`
2. Run `mkdocs serve`
3. Visit the URL given, appending URL's from the [old list of URL's](https://gist.github.com/Jonty/09fa57685e4eef1c54398660f0a563df) to the end. You'll need to strip the `/en/latest` bit, as that is added by RTD and won't work with `mkdocs`
4. Check that every URL in the [old list of URL's](https://gist.github.com/Jonty/09fa57685e4eef1c54398660f0a563df) is redirected to the appropriate page.

## Who can review

Anyone, but @benh-gds should probably have a say if this approach is ok.
